### DISCUSSION TITLE
Tracking stimuli across jspsych sessions

### DISCUSSION THREAD

**Discussion by jaszhou1** [jspsych/jsPsych#1622] [Q&A]
Hi jspsych community!

I want to reuse stimuli for a memory task across multiple experimental sessions, but I don't want repeat items until every item has been used once (and no third uses before all items are used twice etc.) 

Does anyone know of a good way to implement this?

**Comment by cslongman** [comment]
Could you do all of the randomisation and allocation of stimului to conditions (etc.) for each participant offline and then refer to this database/spreadsheet in your script? You could then use URL variables to let the script know the participant and session number.

There may be a more efficient way of doing it, but I think that one way or another you are going to have do some stuff offline (or pass information from the end of one session to the start of another) manually.

Hope this help.
